tests/component/test_python_component.py

```diff
@@ -0,0 +1,32 @@
+import dask.dataframe as dd
+import pandas as pd
+import pyarrow as pa
+from fondant.component import DaskLoadComponent, python_component
+from fondant.pipeline import Pipeline
+
+
+def test_python_component():
+    pipeline = Pipeline(name="dummy-pipeline", base_path="./data")
+
+    @python_component(
+        base_image="python:3.8-slim-buster",
```

Comment:
Here we'll need to provide a base docker image with `Fondant` installed, right?

Reply:
yes we need a sane default which we ideally configure in 1 place --> here